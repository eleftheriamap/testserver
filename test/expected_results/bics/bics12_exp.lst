
test/exp/test/gen/bics/bics12_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	91506d29 	add	x9, x9, #0x41b, lsl #12
   4:	51715f39 	sub	w25, w25, #0xc57, lsl #12
   8:	6af90924 	bics	w4, w9, w25, ror #2
   c:	8a000000 	and	x0, x0, x0
