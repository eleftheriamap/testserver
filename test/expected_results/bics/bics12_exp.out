Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000011000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 000000000041b000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 00000000ff3a9000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : ----
Non-zero memory:
0x00000000 : 0x91506d29
0x00000004 : 0x51715f39
0x00000008 : 0x6af90924
0x0000000c : 0x8a000000
